### PR TITLE
activates blog again, but latest news is not shown

### PR DESCRIPTION
**Pull request by alosdev**
I wrote two little blog posts based on the ukraine ones. But unfortunately the latest news on frontpage does not show up. If someone can help me, I would be happy.

### CODE REVIEW

File: _posts/2015-09-18-call-for-participation.markdown [hunk 1]
```diff
@@ -23,11 +23,12 @@ Our team prepared set of interesting topics in but if you have any cool idea you
 __Google I/O 2014__
```
Comment (mariobodemann) [inline _posts/2015-09-18-call-for-participation.markdown]: This should be changed to `2015`.

File: _posts/2015-09-18-call-for-participation.markdown [hunk 2]
```diff
@@ -53,15 +56,15 @@ __General__
...
   * Storages (SQL, NoSQL, BigData)
   * APIs
-* Go (as a language for a Cloud Ready Apps)
+* Go
 
 
-#### Please submit your proposals [here](http://bit.ly/dfua-c4p).
-__Deadline__ is September 1, 2014
+#### Please submit your proposals [here](https://conference.devfest-berlin.de/en/devfestberlin2015/cfp/session/new).
+__Deadline__ is mod October, 2015
```
Comment (mariobodemann) [inline _posts/2015-09-18-call-for-participation.markdown]: I think `mod` should be `mid`.

File: _config.yml [hunk 3]
```diff
@@ -198,8 +199,8 @@ footerBlocks:
 
 # Blog
 blogTitle: "Latest news"
```
Comment (mariobodemann) [inline _config.yml]: I know, not your change, but should we capitalize `news`, as in `Latest News`?

**Comment by mariobodemann** [comment]
To answer your question: I have no idea, why it didn't show up. But I found some typos instead ... ;)

**Comment by stefanhoth** [comment]
I know why it doesn't work. The section is commented out in index.html. I also don't know if there's a page at /blog. Did you try this?

**Comment by alosdev** [comment]
OK, thanks, than I will fix that asap

File: _posts/2015-09-18-call-for-participation.markdown (hunk 2 above)
Comment (alosdev) [inline _posts/2015-09-18-call-for-participation.markdown]: done

File: _config.yml (hunk 3 above)
Comment (alosdev) [inline _config.yml]: will change it

**Pull request merged**